src/Meziantou.Analyzer/Rules/MethodOverridesShouldNotChangeParameterDefaultsAnalyzer.cs

```diff
@@ -10,8 +10,8 @@ public sealed class MethodOverridesShouldNotChangeParameterDefaultsAnalyzer : Di
 {
     private static readonly DiagnosticDescriptor s_rule = new(
         RuleIdentifiers.MethodOverridesShouldNotChangeParameterDefaults,
-        title: "Method overrides should not change parameter defaults",
-        messageFormat: "Method overrides should not change parameter defaults (original: {0}; current: {1})",
+        title: "Method overrides should not change default arguments",
```

Comment:
I think we should use "default values" as in the documentation https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/named-and-optional-arguments?WT.mc_id=DT-MVP-5003978#optional-arguments

Reply:
Done